docs/guides/Getting-Started/orders-overview.md

```diff
@@ -68,19 +68,19 @@ The Orders Feed is a list of order updates, meaning that whenever there is an ev
 
 ### Hook
 
-The Hook is a complement to the Feed, which allows an integration to consume order updates data by sending items to an URL provided by the user in the Hook configuration.
+Hook is a counterpart to Feed, which allows an integration to get order update data by sending items to a URL provided by the user in the hook configuration.
```

Comment:
```suggestion
Hook works as a complement to Feed, because it allows an integration to consume order updates data by sending items to a URL provided by the user in the Hook configuration.
```